mesonbuild/scripts/meson_install.py

```diff
@@ -121,23 +150,24 @@ def do_install(datafilename):
     d.destdir = os.environ.get('DESTDIR', '')
     d.fullprefix = destdir_join(d.destdir, d.prefix)
 
-    install_subdirs(d) # Must be first, because it needs to delete the old subtree.
-    install_targets(d)
-    install_headers(d)
-    install_man(d)
-    install_data(d)
-    run_install_script(d)
+    d.dirmaker = DirMaker()
+    with d.dirmaker as dm:
```

Comment:
__[flake8]__
* [F841] local variable 'dm' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->